spec/controllers/games_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe GamesController, type: :controller do
```

Comment:
There should also be tests that verify that a logged in user can create, join, and view games.